From e116bd43c8cd1f7eb388ce87b8b2df5ca548a710 Mon Sep 17 00:00:00 2001
From: Arun R Murthy <arun.r.murthy@intel.com>
Date: Wed, 11 Nov 2015 09:06:53 +0530
Subject: [PATCH 28/28] SQUASHME [VPG]: drm/i915: add proper print level for
 pfit

Upon calculating the aspect ratio and detecting the mode for
the panel fitter to be enabled, if there is a discrepancy
in the mode requested from the user space, instead of printing
error, just print an info and proceed with the calculated mode

This patch has to be squashed with
commit 4804b54d8cdc0ce2a0a6b31ca49b533b3b30924b
Author: Arun R Murthy <arun.r.murthy@intel.com>
Date:   Tue Aug 11 14:29:54 2015 +0530

    SQUASHME! [VPG]: calculate the aspect ratio for pfit

Change-Id: Iab21192232ac1313051d1f2ea82cb2bb6adf6cf8
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-8771
Signed-off-by: Arun R Murthy <arun.r.murthy@intel.com>
Reviewed-on: https://android.intel.com:443/437349
---
 drivers/gpu/drm/i915/intel_display.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 444e071..d5b163a 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -11288,7 +11288,7 @@ int intel_set_disp_calc_flip(struct drm_mode_set_display *disp,
 			}
 
 			if (pfit_mode != (pfit_control & ~MASK_PFIT_SCALING_MODE)) {
-				DRM_ERROR("pfit mode(%d) mismatch with calculated mode(%d)\n",
+				DRM_INFO("pfit mode(%d) mismatch with calculated mode(%d)\n",
 						(pfit_control & ~MASK_PFIT_SCALING_MODE), pfit_mode);
 				pfit_control &=  MASK_PFIT_SCALING_MODE;
 				pfit_control |= pfit_mode;
-- 
1.9.1

